### PR TITLE
call filelock function

### PR DESCRIPTION
**Pull request by pramenku**
call install_filelock before install_triton

To fix https://ontrack-internal.amd.com/browse/SWDEV-384803

Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by pruthvistony — CHANGES_REQUESTED** [review]
Was hesitant to do this change, may be ok for 5.5 but 5.6 needs to be reverted after the IFU.
since it is NOT proper way.

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -922,6 +925,7 @@ elif [[ "${SHARD_NUMBER}" == 2 && $NUM_TEST_SHARDS -gt 1 ]]; then
   test_torch_function_benchmark
 elif [[ "${SHARD_NUMBER}" -gt 2 ]]; then
   # Handle arbitrary number of shards
+  install_filelock
```
[the remark this answers is not in the document]
Comment (pramenku) [reply .jenkins/pytorch/test.sh]: Removed L928

**Comment by pramenku** [comment]
> Was hesitant to do this change, may be ok for 5.5 but 5.6 needs to be reverted after the IFU.
Sure, @pruthvistony  we can revert after IFU merge for 5.6. Till that time, at least we will able to test those module and report any new bug.

**Comment by pramenku** [comment]
Hi @pruthvistony can you please merge it today in 5.5 n 5.6.

We couldn't test these modules in automation until this PR is merged.

Thanks

**Review by pruthvistony — APPROVED** [review]

**Pull request merged**